components/Archetype/basic_usage.rst

```diff
@@ -0,0 +1,58 @@
+Basic Usage
+===========
+
+The usage of this component bases on a few classes cooperation.
+We create an **Archetype** with given **Attributes** as an *ArrayCollection*.
+To create an **Archetype** we can use its Builder class, which will use
+our **AttributeValues** repository.
+
+Archetype
+---------
+
+.. code-block:: php
+
+    <?php
+
+    use Doctrine\Common\Collections\ArrayCollection;
+    use Sylius\Component\Attribute\Model\Attribute;
+    use Sylius\Component\Archetype\Model\Archetype;
+    use Sylius\Component\Archetype\Builder\ArchetypeBuilder;
+
+    $archetype = new Archetype();
+
+    // Before you can start using the new archetype please set its current and fallback locales.
+    $archetype->setCurrentLocale('en');
+    $archetype->setFallbackLocale('en');
+
+    // Let's create an attribute for our archetype
+    $attribute = new Attribute();
+    $attribute->setName('Mug material');
+    $attribute->setType('text');
+
+    // And then let's add it to a collection
+    $attributes = new ArrayCollection();
+    $attributes->add($attribute);
+
+    $archetype->setName('Mug');
+    $archetype->setAttributes($attributes);
+    $archetype->getName(); // returns 'Mug'
+    $archetype->getAttributes(); // returns $attributes
+
+
+ArchetypeBuilder
+----------------
+
+.. code-block:: php
+
+    /**
+     * @param RepositoryInterface $attributeValueRepository
+     */
+    $builder = new ArchetypeBuilder($attributeValueRepository);
+
+    /**
+     * @var ArchetypeSubjectInterface $subject
+     */
+    $subject = /* ... */;
```

Comment:
Well in that case it isn't really usefull. The class implements a lot of methods, it would be really long. And I would use it just to show its 'usage' here. Imo.